Dockerfile

```diff
@@ -34,20 +34,29 @@ RUN locale-gen en_US.UTF-8
 # configured in /etc/default/locale so we need to set it manually.
 ENV LC_ALL=en_US.UTF-8
 
-RUN adduser --gecos '' --disabled-password coder && \
+RUN addgroup --gid 1000 coder && \
```

Comment:
by default the first user to be made in Docker build is 1000 already, I don't see the need for extra steps.

Reply:
> by default the first user to be made in Docker build is 1000 already

This is false. Docker does not enforce a default uid of 1000. The Node images that are being used as a base image however [do create a `node` user of 1000:1000](https://github.com/nodejs/docker-node/blob/f5875531604b4b3b9fbc36437182781c3655c8ae/10/buster/Dockerfile#L3).

That said, this doesn't need to create a new user for 1000:1000, I'm not sure what is up with all the other options in addition to that. The intention appears to be replacing `node` with `coder` which is used later on.  Looks like they're referencing this addition from [`fixuid` installation docs](https://github.com/boxboat/fixuid#install-fixuid-in-dockerfile) without much thought..